dll/win32/imm32/imm.c

```diff
@@ -1017,7 +1017,9 @@ LPVOID APIENTRY Imm32HeapAlloc(DWORD dwFlags, DWORD dwBytes)
 {
     if (!g_hImm32Heap)
     {
-        g_hImm32Heap = GetProcessHeap(); // FIXME: Use TEB
+        PTEB pTeb = NtCurrentTeb();
+        PPEB pPeb = pTeb->NtTib.ArbitraryUserPointer;
+        g_hImm32Heap = pPeb->ProcessHeap;
```

Comment:
What's the problem with this fixme? Otherwise the correct thing is:
```suggestion
        g_hImm32Heap = RtlGetProcessHeap();
```

Reply:
Used `RtlGetProcessHeap`. OK.